### PR TITLE
add Safetensors

### PR DESCRIPTION
**Pull request by zhimin-z**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This repository contains a curated list of awesome open source libraries that wi
 | [🧵 Data Pipeline](#data-pipeline) | [🏷️ Data Labelling](#data-labelling) | [📅 Metadata Management](#metadata-management) |
 | [🗺️ Computation Distribution](#computation-load-distribution) | [📥 Model Serialisation](#model-serialisation) | [🧮 Optimized Computation](#optimized-computation)|
 | [💸 Data Stream Processing](#data-stream-processing) | [:red_circle: Outlier & Anomaly Detection](#outlier-and-anomaly-detection) | [🎁 Feature Store](#feature-store) |
-| [⚔ Adversarial Robustness](#adversarial-robustness) | [💾 Data Storage Optimization](#data-storage-optimisation) | [📓 Data Science Notebook](#data-science-notebook) |
+| [⚔ Adversarial Robustness](#adversarial-robustness) | [💾 Data Storage Optimization](#data-storage-optimization) | [📓 Data Science Notebook](#data-science-notebook) |
```
Comment (axsaucedo) [inline README.md:22]: Using UK spelling

**Review by axsaucedo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -610,7 +611,7 @@ This repository contains a curated list of awesome open source libraries that wi
...
 * [Recommenders](https://github.com/Microsoft/Recommenders) ![](https://img.shields.io/github/stars/Microsoft/Recommenders.svg?style=social) - Recommenders contains benchmark and best practices for building recommendation systems, provided as Jupyter notebooks.
 * [RLeXplore](https://github.com/yuanmingqi/rl-exploration-baselines) ![](https://img.shields.io/github/stars/yuanmingqi/rl-exploration-baselines.svg?style=social) - RLeXplore provides stable baselines of exploration methods in reinforcement learning
-* [SafePO-Baselines](https://github.com/PKU-MARL/Safe-Policy-Optimization) ![](https://img.shields.io/github/stars/PKU-MARL/Safe-Policy-Optimization.svg?style=social) - SafePO-Baselines is a benchmark repository for safe reinforcement learning algorithms.
+* [SafePO-Baselines](https://github.com/PKU-MARL/Safe-Policy-Optimisation) ![](https://img.shields.io/github/stars/PKU-MARL/Safe-Policy-Optimisation.svg?style=social) - SafePO-Baselines is a benchmark repository for safe reinforcement learning algorithms.
```
Comment (axsaucedo) [inline README.md:614]: It seems this would break

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Added another comment - would also be worth extending the title so all the libraries added are shown in the release notes

**Comment by zhimin-z** [comment]
Is it good to ship? @axsaucedo

**Review by axsaucedo — APPROVED** [review]
There are some changes that appear as edits strangely, but we're doing squash commit so it should be OK

**Pull request merged**